### PR TITLE
Generate data

### PR DESCRIPTION
**Pull request by courtchao**

### CODE REVIEW

File: db/database.js [hunk 1]
```diff
@@ -11,6 +10,32 @@ const connection = mysql.createConnection({
 
 connection.connect();
 
+connection.query('CREATE DATABASE IF NOT EXISTS reviews;', (error) => {
+  if (error) {
+    throw error;
+  }
```
Comment (cdl08001) [inline db/database.js]: Maybe throw a console log in here to indicate when the db was created?

File: db/database.js [hunk 2]
```diff
@@ -11,6 +10,32 @@ const connection = mysql.createConnection({
...
+  date DATE NOT NULL,
+  text VARCHAR(2000),
+  source VARCHAR(40) NOT NULL,
+  item_id INT NOT NULL,
+  PRIMARY KEY (id)
+  );`, (error) => {
+  if (error) {
+    throw error;
+  }
+});
```
Comment (cdl08001) [inline db/database.js]: Same thing here, having some indication that a table was actually created might be helpful.

File: db/database.js [hunk 3]
```diff
@@ -13,22 +13,24 @@ connection.connect();
...
+  for (let i = 0; i < randomNumber; i += 1) {
     const rating = Math.floor(Math.Random() * 5);
     const title = faker.lorem.sentence();
     const date = faker.date.past();
     const text = faker.lorem.paragraphs();
     const source = sources[Math.floor(Math.Random() * 5)];
-    const query = 'INSERT INTO itemReviews (id, rating, title, date, text, source) VALUES (null, ? ? ? ? ?)';
-    connection.query(query, [rating, title, date, text, source], (err, results) => {
+    const item = itemId;
+    const query = 'INSERT INTO itemReviews (id, rating, title, date, text, source, item_id) VALUES (null, ? ? ? ? ? ?)';
```
Comment (cdl08001) [inline db/database.js]: Does the '? ? ? ? ? ?' you have on this line work? They don't need to be separated by commas? I'm curious more than anything haha

File: db/database.js (hunk 2 above)
Comment (courtchao) [reply db/database.js]: Thanks for the feedback Cal! I'll make the changes now

**Pull request merged**